os/board/common/crashdump.c

```diff
@@ -156,6 +162,38 @@ static int ramdump_via_uart(void)
 		}
 	}
 
+	/* Will dump Filesystem contents */
+	up_lowputc('F');
+
+	/* Receive hanshake string from HOST */
+        do {
+                host_buf[0] = up_getc();
+        } while (host_buf[0] != target_str[0]);
+
+        for (i = 1; i < strlen(target_str);) {
+                if ((ch = up_getc()) != -1) {
+                        host_buf[i] = ch;
+                        i++;
+                }
+        }
+
+        if (strncmp(host_buf, FSDUMP_HANDSHAKE_STRING, HANDSHAKE_STR_LEN_MAX) != 0) {
+                /* Send NAK */
+                up_lowputc('N');
+        }
+
+filesystem_dump:
+        /* Send ACK */
+        up_lowputc('A');
+
+	ptr = (uint8_t *)0x04620000;
```

Comment:
Why is teh address and size hardcoded ?
I feel it is better to use CONFIG values